### ISSUE TITLE
create-react-app doesn't work with ppm

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#444]
`create-react-app` is the standard way to start a React project now and pnpm does not work with it.

The fix was added to `enhanced-resolver` https://github.com/webpack/webpack/issues/2937 in v2.3. `create-react-app` uses `webpack@1.13.2` which uses  `enhanced-resolve@~0.9.0"`.
### Code to reproduce the issue:

```
create-react-app foo
cd foo
rm -rf node_modules
pnpm install
npm start
```
### Expected behavior:

App starts without errors.
### Actual behavior:

```
Error in /Users/Vaughan/.pnpm/.store/nested/react-dom@15.3.2/_/index.js
Module not found: 'react/lib/ReactDOM' in /Users/Vaughan/.pnpm/.store/nested/react-dom@15.3.2/_

 @ /Users/Vaughan/.pnpm/.store/nested/react-dom@15.3.2/_/index.js 3:17-46
```
### Additional information:
- `node -v` prints: 6.5.0
- Windows, OS X, or Linux?: macOS

**Comment by vjpr** [comment]
I have requested a backport of the functionality in `enhanced-resolve`.

**Comment by vjpr** [comment]
A simple fix for `create-react-app` is to add:

```
resolve: {
  root: path.join(process.cwd(), 'node_modules'),
}
```

They already have the fallback in there set to `node_modules` - not sure why it doesn't work...

**Comment by zkochan** [comment]
Seems to work with the next version of pnpm without any tweaks

Try `npm i -g pnpm@next`

**Comment by zkochan** [comment]
@vjpr I just published v0.42.6 as latest and I can't reproduce this issue there. Although I was able to reproduce it on earlier versions of pnpm.

**Comment by zkochan** [comment]
This should work on pnpm 0.42.6 - 0.50. Now there's a new issue #581
